### ISSUE TITLE
Support for latest Podman versions?

### ISSUE THREAD

**Issue by jackgraddon** [minecraft-podman/podcraft#14]
Will podcraft ever support later versions of podman, including the latest version hosted on the Ubuntu Archive, 3.4.4?

**Comment by AstraLuma** [comment]
whoa, people are interested in this?

Honestly, I'm not actively using or working on this.

iirc, I wrote this when podman was using the old API before they switched to the Docker-like API.

**Comment by jackgraddon** [comment]
This just seems like the easiest way for people to run minecraft servers in podman! No worries if there is no plan to support it any further. Thanks for responding!

**Comment by AstraLuma** [comment]
Yeah, it was my attempt at a super-helpful, super mature, containerized minecraft setup.

If you want something easier to maintain, see if podman-compose works for your needs?

**Comment by AstraLuma** [comment]
Oh, and a lot of the described features are more aspirational--I never implemented a bunch of the stuff I would have liked to.
